src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/ListCollector.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public class ListCollector<T> : ICollector<T>
```

Comment:
I'm not a fan of bleeding the underlying implementation thru by naming. Could we do
```suggestion
    public class Collector<T> : ICollector<T>
```